pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -176,6 +176,8 @@ public void testHardcodedSqlQueries()
     testSqlQuery(query, Collections.singletonList(query));
     query = "SELECT DaysSinceEpoch, COUNT(*), MAX(ArrTime), MIN(ArrTime) FROM mytable GROUP BY DaysSinceEpoch";
     testSqlQuery(query, Collections.singletonList(query));
+    query = "SELECT ArrTime, ArrTime * 10 FROM mytable WHERE DaysSinceEpoch >= 16312";
```

Comment:
could we add some more queries here which will cover all the arithmetic operators?

Reply:
added

Reply:
Can we add tests for nested? e.g ADD(col1 + 5, col2), ADD(col1 * 5, 2)